src/Bridges/vaftosafbridge.jl

```diff
@@ -0,0 +1,148 @@
+end
+
+function MOI.canget(bridge::VectorToScalarBridge{T, S},
+       ::MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{T}, S}) where {T, S}
+    return true
+end
+function MOI.get(bridge::VectorToScalarBridge{T, S},
+       ::MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{T}, S}) where {T, S}
+    return bridge.constraints
+end
+
+function MOI.delete!(model::MOI.ModelLike, bridge::VectorToScalarBridge)
+    MOI.delete!.(Ref(model), bridge.constraints)
+end
+
+function MOI.canget(model::MOI.ModelLike, attribute::MOI.ConstraintDual,
+                    ::Type{VectorToScalarBridge{T, S}}) where {T, S}
+    return MOI.canget(model, attribute, CI{MOI.ScalarAffineFunction{T}, S})
+end
+function MOI.get(model::MOI.ModelLike, attribute::MOI.ConstraintDual,
+                 bridge::VectorToScalarBridge)
+    return MOI.get.(Ref(model), Ref(attribute), bridge.constraints)
+end
+
+function MOI.canget(model::MOI.ModelLike, attribute::MOI.ConstraintPrimal,
+                    ::Type{VectorToScalarBridge{T, S}}) where {T, S}
+    return MOI.canget(model, attribute, CI{MOI.ScalarAffineFunction{T}, S}) &&
+        MOI.canget(model, MOI.ConstraintSet(), CI{MOI.ScalarAffineFunction{T}, S})
+end
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintPrimal, bridge::VectorToScalarBridge)
+    primals = fill(0.0, length(bridge.constraints))
+    for (i, index) in enumerate(bridge.constraints)
+        primal = MOI.get(model, MOI.ConstraintPrimal(), index)
+        set = MOI.get(model, MOI.ConstraintSet(), index)
+        set_constant = MOIU.getconstant(set)
+        primals[i] = primal - set_constant
+    end
+    return primals
+end
+
+function MOI.modify!(model::MOI.ModelLike, bridge::VectorToScalarBridge{T, S},
+                     change::MOI.VectorConstantChange{Float64}) where {T, S}
+    @assert length(bridge.constraints) == length(change.new_constant)
+    for (index, coef) in zip(bridge.constraints, change.new_constant)
+        MOI.set!(model, MOI.ConstraintSet(), index, S(-coef))
+    end
+end
+
+function MOI.modify!(model::MOI.ModelLike, bridge::VectorToScalarBridge{T, S},
+                     change::MOI.MultirowChange{T}) where {T, S}
+    @assert length(bridge.constraints) == length(change.new_coefficients)
+    variable = change.variable
+    for (row, coef) in change.new_coefficients
+        index = bridge.constraints[row]
+        MOI.modify!(model, index, MOI.ScalarCoefficientChange(variable, coef))
+    end
+end
+
+MOI.canget(::MOI.ModelLike, ::MOI.ConstraintSet, ::Type{<:VectorToScalarBridge}) = true
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintSet,
+                 bridge::VectorToScalarBridge{T, S}) where {T, S}
+    return get_vector_set(S)
+end
+
+MOI.canget(::MOI.ModelLike, ::MOI.ConstraintFunction, ::Type{<:VectorToScalarBridge}) = true
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintFunction,
+                 bridge::VectorToScalarBridge{T, S}) where {T, S}
+    terms = VectorAffineTerm{T}[]
```

Comment:
`moivcat` can be helpful here. Using `moivcat` will help adding support for quadratic functions